client/driver/docker.go

```diff
@@ -99,18 +99,6 @@ type DockerDriverConfig struct {
 	WorkDir          string              `mapstructure:"work_dir"`           // Working directory inside the container
 }
 
-// Validate validates a docker driver config
-func (c *DockerDriverConfig) Validate() error {
```

Comment:
I understand why the `c.ImageName` validation was in `validate()`. But are there any possibility that the image would be disappeared from `DockerDriverConfig` after the decoding validation?

As for following two lines are not validations. So, I think they should be moved to `NewDockerDriverConfig`.

~~~
        c.PortMap = mapMergeStrInt(c.PortMapRaw...)
        c.Labels = mapMergeStrStr(c.LabelsRaw...)
~~~